mowgli_etl/pipeline/wdc/wdc_extractor.py

```diff
@@ -0,0 +1,17 @@
+from typing import Dict, Optional
+
+from pathvalidate import sanitize_filename
+
+from mowgli_etl._extractor import _Extractor
+from mowgli_etl.pipeline.wdc.wdc_constants import WDC_CSV_FILE_KEY
+from mowgli_etl.pipeline_storage import PipelineStorage
+from mowgli_etl.paths import DATA_DIR
+
+class WdcExtractor(_Extractor):
+	_WDC_ARCHIVE_PATH = DATA_DIR / "wdc" / "extracted" / "offers_corpus_english_v2_random_100.jsonl"
```

Comment:
You don't need to hard-code this, just go off storage.extracted_data_dir_path, which points to the same directory.